.github/workflows/test_notebooks.yaml

```diff
@@ -0,0 +1,53 @@
+name: test_notebooks
+
+on: [pull_request]
+
+jobs:
+  run_notebooks:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.9"]
```

Comment:
this can be removed - matrix is useful for when you have several configs you want to test for, a matrix with a single item is the same as just using that item directly

Reply:
I know but I think it's better to define this in one single place (it's used multiple times). Also, at some point it could be useful to test more versions.